### PR TITLE
Cycle orm v2

### PR DESCRIPTION
**Pull request by roxblnfk**
| Q             | A
| ------------- | ---
| New feature?  | ✔️

- Use attributes instead of annotations
- use ORM v2 instead of v1
  - cycle/database v2
  - DTO connection config
  - Behaviors instead of custom mappers

Some fixes

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -21,15 +21,16 @@
...
-    $c3 = dirname(__DIR__) . '/c3.php';
+    $c3 = 'c3.php';
     if (file_exists($c3)) {
         require_once $c3;
     }
 }
 
 // Run HTTP application runner
-$runner = new HttpApplicationRunner(dirname(__DIR__), $_ENV['YII_DEBUG'], $_ENV['YII_ENV']);
+$runner = new HttpApplicationRunner(getcwd(), $_ENV['YII_DEBUG'], $_ENV['YII_ENV']);
```
Comment (samdark) [inline public/index.php:35]: What is the reason for this change?

Comment (roxblnfk) [reply public/index.php:35]: try to compare current working dir in cases:
- `php yii serve`
- `php -S 127.0.0.1:8080 -t public` (like in [tests](https://github.com/yiisoft/demo/blob/15504bd4383685f2a095e857448d5c274315fab0/.github/workflows/build.yml#L65))

**Comment by rustamwin** [comment]
#418

File: public/index.php (hunk 1 above)
Comment (samdark) [reply public/index.php:35]: That won't work if you'll launch `./yii` not from the same directory as it is in:

```sh
./apps/yii3_demo/yii serve
```

Comment (roxblnfk) [reply public/index.php:35]: > That won't work

Feel free to fix it

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍
